src/ripple/server/impl/BaseWSPeer.h

```diff
@@ -43,6 +43,12 @@ class BaseWSPeer
     using BasePeer<Impl>::fail;
     using BasePeer<Impl>::strand_;
 
+    enum
+    {
+        // Max seconds without completing a message
+        timeoutSeconds = 30
```

Comment:
Or `auto constexpr timeout = 30s;` ?

That requires a `using namespace `std::chrono_literals` in a header, but we've already gone there for ripple (an application), but not beast (a library).